### PR TITLE
Remove TheftAlert state management logic

### PR DESCRIPTION
**Pull request by jmromer**
Exposes more granular control via the admin panel

Screen Shot 2019-06-24 at 6 07 36 PM
Screen Shot 2019-06-24 at 6 07 51 PM

### CODE REVIEW

File: app/views/admin/theft_alerts/edit.html.haml [hunk 1]
```diff
@@ -6,7 +6,18 @@
...
     html: { class: "m-0" } do |f|
-    = hidden_field_tag :state_transition, params.fetch(:state_transition, "update_details")
+    .form-group
+      = f.label :status, class: "form-label"
+      = f.select :status, options_for_select(TheftAlert.statuses.keys), class: "form-control"
+    .form-group
+      = f.label :theft_alert_plan_id, class: "form-label"
+      = f.select :theft_alert_plan_id, options_for_select(TheftAlertPlan.active.map{|e| [e.title_string, e.id]}), class: "form-control"
+    .form-group
+      = f.label :begin_at, class: "form-label"
```
Comment (sethherr) [inline app/views/admin/theft_alerts/edit.html.haml]: A few requests:

1. Can you automatically set the `begin_at` when the alert is switched to begin, and set the `end_at` based on that?
2. Can you hide the times and only show them after the alert is enabled?
3. Can you use `datetime_local_field`, include a timezone hidden field, and use `TimeParser` to set the time?

File: app/views/admin/theft_alerts/edit.html.haml [hunk 2]
```diff
@@ -6,7 +6,18 @@
     url: admin_theft_alert_path(@theft_alert),
     method: :patch,
     html: { class: "m-0" } do |f|
-    = hidden_field_tag :state_transition, params.fetch(:state_transition, "update_details")
+    .form-group
+      = f.label :status, class: "form-label"
+      = f.select :status, options_for_select(TheftAlert.statuses.keys), class: "form-control"
+    .form-group
+      = f.label :theft_alert_plan_id, class: "form-label"
```
Comment (sethherr) [inline app/views/admin/theft_alerts/edit.html.haml]: Do you think there is a reason admins need to be able to edit this?

Comment (jmromer) [reply app/views/admin/theft_alerts/edit.html.haml]: A scenario in which someone requests a plan change (potentially with accompanying changes to start/end time fields) is more plausible than changes being required to, say, `creator` or `stolen_record`.

Comment (sethherr) [reply app/views/admin/theft_alerts/edit.html.haml]: Fair. I can see the use of it for that. I also don't see a reason to enable editing creator or stolen_record

File: app/views/admin/theft_alerts/edit.html.haml (hunk 1 above)
Comment (jmromer) [reply app/views/admin/theft_alerts/edit.html.haml]: Updated in  e476085 and 0ecd791

**Comment by jmromer** [comment]
👍 Added in 30d0034c

**Pull request merged**